core/src/main/resources/lib/layout/keyboard-shortcut.jelly

```diff
@@ -0,0 +1,43 @@
+<!--
+The MIT License
+
+Copyright (c) 2023, Jenkins Contributors
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define">
+  <st:documentation>
+    <st:attribute name="message">
+      Optional message for the shortcut
```

Comment:
```suggestion
      Tooltip message for the shortcut. Use the placeholder <code>{0}</code> escaped (wrapped in <code>&quot;</code>). A minimal message consisting of only the shortcut would be <code>&quot;{0}&quot;</code>.
```

(Assuming `<code>` works here.)

Reply:
it renders the code and &quot literally with the intellij plugin, unsure about stapler-maven-plugin itself, searching the code I couldn't see any handling of it though.